src/OpenConext/UserLifecycle/Domain/ValueObject/Client/ErrorMessage.php

```diff
@@ -30,6 +34,14 @@ class ErrorMessage
      */
     public function __construct($errorMessage = null)
     {
+        // Setting multiple error messages is supported, but only if all array entries are of type string
+        if (is_array($errorMessage)) {
+            foreach ($errorMessage as $message) {
+                if (!is_string($message)) {
+                    throw new InvalidArgumentException('All of the error messages must be of type string');
+                }
+            }
+        }
```

Comment:
At this point $errorMessage could still be an int if the original $errorMessage is an int.
This is as intended?

Reply:
A good observation. But I'm leaving it as is. In your example, the integer error message would not cause an error down the line as it can and will be casted to be a string. The message would not make any sense, but hey..